includes/admin/settings/display-settings.php

```diff
@@ -21,12 +21,86 @@ function edd_admin_header() {
 	if ( ! edd_is_admin_page( '', '', false ) ) {
 		return;
 	}
-
 	$numberNotifications = EDD()->notifications->countActiveNotifications();
+	$current_page        = ! empty( $_GET['page'] ) ? $_GET['page'] : '';
+	$is_single_view      = ! empty( $_GET['view'] );
+
+	$page_title = '';
+	switch( $current_page ) {
+		case 'edd-settings':
+			$page_title = __( 'Settings', 'easy-digital-downloads' );
+			break;
+		case 'edd-reports':
+			$page_title = __( 'Reports', 'easy-digital-downloads' );
+			break;
+		case 'edd-payment-history':
+			$page_title = __( 'Orders', 'easy-digital-downloads' );
+			break;
+		case 'edd-discounts':
+			$page_title = __( 'Discounts', 'easy-digital-downloads' );
+			break;
+		case 'edd-customers':
+			$page_title = __( 'Customers', 'easy-digital-downloads' );
+			break;
+		case 'edd-tools':
+			$page_title = __( 'Tools', 'easy-digital-downloads' );
+			break;
+		default:
+			if ( ! empty( $_GET['page'] ) ) {
+				$page_title = ucfirst( str_replace( array( 'edd-', 'fes-' ), '', $current_page ) );
+			} else {
+				if ( ! empty( $_GET['post_type'] ) ) {
+					$post_type  = get_post_type_object( $_GET['post_type'] );
+					$page_title = $post_type->labels->name;
+				} else {
+					$page_title = __( 'Downloads', 'easy-digital-downloads' );
+				}
+			}
+
+			break;
+	}
+
+	$page_title = apply_filters( 'edd_settings_page_title', $page_title, $current_page );
+	if ( ! empty( $page_title ) && empty( $is_single_view ) ) {
+		?>
+		<style>
+			.wrap > h1,
+			.wrap h1.wp-heading-inline,
+			.wrap h2,
+			.page-title-action {
+				display: none;
+			}
+		</style>
+		<script>
+		jQuery(document).ready(function($){
+			const coreAddNew = $('.page-title-action');
+			const eddAddNew  = $('.add-new-h2');
+
+			if ( coreAddNew.length ) {
+				coreAddNew.appendTo('.edd-header-page-title-wrap').addClass('button').show();
+			}
+
+			if ( eddAddNew.length ) {
+				eddAddNew.appendTo('.edd-header-page-title-wrap').addClass('button');
+			}
+		});
+		</script>
+		<?php
+	}
+
 	?>
 	<div id="edd-header" class="edd-header">
 		<div id="edd-header-wrapper">
-			<img class="edd-header-logo" alt="" src="<?php echo esc_url( EDD_PLUGIN_URL . '/assets/images/logo-edd-dark.svg' ); ?>" />
+			<span id="edd-header-branding">
+				<img class="edd-header-logo" alt="" src="<?php echo esc_url( EDD_PLUGIN_URL . '/assets/images/logo-edd-dark.svg' ); ?>" />
+			</span>
+
+			<?php if ( ! empty( $page_title ) ) : ?>
+			<span class="edd-header-page-title-wrap">
+				<span class="edd-header-separator">/</span>
+				<h1 class="edd-header-page-title"><?php echo esc_html( $page_title ); ?></h1>
```

Comment:
Probably this needs to be an `h1` or `span` conditionally, maybe, depending on if the `$single_view` variable is true. On the individual customer details screen, or add/edit orders, now I have two `h1` and that's not ideal either.

Reply:
@robincornett Done.